package.json

```diff
@@ -56,18 +59,19 @@
         "glob": "7.1.4",
         "hapi-openapi": "1.2.2",
         "hapi-swagger": "10.0.2",
-        "immutable": "^4.0.0-rc.12",
+        "immutable": "4.0.0-rc.12",
         "jsdoc": "3.6.2",
         "mustache": "3.0.1",
         "uuid4": "1.1.4"
     },
     "devDependencies": {
         "ava": "2.1.0",
-        "eslint": "5.16.0",
+        "eslint": "6.0.1",
```

Comment:
I noticed this project uses eslint instead of standard, which all of the other projects use. Is there a reason for that?

Reply:
Nope. Just used eslint as it was a part of the auto generated code. Can add in standard

Reply:
Please do :D